### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by leozide** [leozide/leocad#542] [General]
Moving @nathaneltitane question to a new topic.

> @leozide do you have a roadmap to follow for all fixes and implementation you are writing into the project?

No, I just work on whatever I feel like. My plan was to work on the instructions layout functionality next but I haven't had much time to focus on big tasks lately so I just work on small fixes.

**Comment by nathaneltitane** [comment]
> Moving @nathaneltitane question to a new topic.
> 
> > @leozide do you have a roadmap to follow for all fixes and implementation you are writing into the project?
> 
> No, I just work on whatever I feel like. My plan was to work on the instructions layout functionality next but I haven't had much time to focus on big tasks lately so I just work on small fixes.

I agree and hear you on that, difficult to do personal stuff here too with work taking all the resources.

May I suggest looking at all the small regressions and enhancement tickets as they usually point to some lack of consistency in the implementation of new or improved features? (keyboard shortcuts, indicators, workflow, etc.)

See and manage #521 #518 #517 #516 #514 #513 #508 - i'M no expert but sounds like they're half-way htere ☺

**Comment by rsbx** [comment]
What are the chances for an actual versioned release before the end of the January? The reason I'm curious is that it would be nice to have a more up-to-date LeoCad in Debian 11 (Bullseye, the next release) and it is going to start freezing changes; the cutoff for new code is early February.

**Comment by leozide** [comment]
I'd have to stop making changes for a bit and take some time to test and fix things I may have broken.
